### ISSUE TITLE
adding a language after initial configuration loses language selector widget

### ISSUE THREAD

**Issue by muellert** [plone/plone.app.multilingual#113]
On a new and mostly empty Plone 4.3.2 site, I have plone.app.contenttypes and plone.multilingualbehavior 1.2 installed, and set up with two languages ("de", "en"). IfI add another language, the language selector widget, as well as the "Translate" document action, vanishes. Looking into the PMI in "Dexterity Content Types" shows that the relevant content types have all "Multilingual Behavior" checked. Adding new content does not bring it back, and neither does running the steps in "Site cleanup".

**Comment by GrainesDePaix** [comment]
As a front-end developer and Plone-newbie I experienced a similar issue recently...

Created a clean 4.3.4 site (démo) for a client to view the work we had advanced on. After adding some folders to display the site architecture, a sample files and pages, as well as the addition of two panels using `collective.panels`. Only at that point I decided to active the 9 languages of the site through the Plone control panel, with the assumption that the folders and setup I created would be duplicated into the other 8 languages automatically. I assumed wrongly. 

Result: the language selector and the `collective.panels` were no longer visible on the site - replacing the language selector at the top of the page was a string reading:
>Site démo error while rendering plone.global_sections

Attempted an undo through the ZMI, with little impact, the error persisted. 
Ended up having to redo an reinstall of the website, and after consulting with a Plone developer I understood that language selection has to happen at site creation and that the addition of a new language to `plone.app.multilingual` when content was already present can cause a critical error.

A safe mechanism for the addition of languages (both for the language selector and a Plone site as a whole) after content has been created would be very useful and a welcome enhancement. 

There are plenty of new users which would naively assume as I did that adding a new language would not cause such errors.
